shared/templates/static/bash/file_permissions_sshd_private_key.sh

```diff
@@ -1,3 +1,8 @@
 # platform = multi_platform_rhel
 
-chmod 0640 /etc/ssh/*_key
+#check if files exist before attempting 
+#to change them
+if ls /etc/ssh/*_key &> /dev/null
+then
+    chmod 0640 /etc/ssh/*_key
+fi
```

Comment:
If at boot there isn't any keys, I think it should pass.  The vulnerability does not exist as no keys are present to be vulnerable.